classes/PodsUI.php

```diff
@@ -3000,7 +3000,7 @@ public function manage( $reorder = false ) {
 
 							echo PodsForm::submit_button( $this->header['search'], 'button', false, false, array( 'id' => $this->num_prefix . 'search' . $this->num . '-submit' ) );
 
-							if ( 0 < strlen( $this->search ) ) {
+							if ( NULL === $this->search ) {
```

Comment:
```suggestion
							if ( 0 < strlen( (string) $this->search ) ) {
```

Reply:
We can simplify this and the null check was negating the original conditional here.

Reply:
Nice! Thank you :kiss: